src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/VersionPage/MetadataTabTests.kt

```diff
@@ -72,6 +73,19 @@ class MetadataTabTests: BaseVersionPageTests()
         assertThat(content.select("#git-commit-value").text()).isEqualTo("abc123")
     }
 
+    @Test
+    fun `renders database instances`()
```

Comment:
Should probably include a test for the case where instances are set but git properties are not